state_synchronizer/src/executor_proxy.rs

```diff
@@ -56,9 +62,12 @@ impl ExecutorProxy {
             &config.storage.address,
             config.storage.port,
         ));
+        let peers_with_public_keys = config.network.trusted_peers.get_trusted_consensus_peers();
```

Comment:
hm....not sure if that's correct source of getting validators. Specially in case of reconfiguration
maybe put TODO here